rules.d/meson.build

```diff
@@ -21,10 +21,13 @@ rules = files('''
         75-net-description.rules
         75-probe_mtd.rules
         78-sound-card.rules
-        80-drivers.rules
         80-net-setup-link.rules
 '''.split())
 
+if conf.get('HAVE_KMOD') == 1
+        rules += ['80-drivers.rules']
```

Comment:
I am not familiar to meson, but it seems you need to use `rules += files('80-drivers.rules')` here.

Reply:
@amoskvin, ping

Reply:
Thanks for reminding! I made the change that was suggested, and semaphoreci passes now, but now it's failing on bionic-i386. I can't tell what the issue is...